src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,57 @@
+package duke.task;
+
+/**
+ * Represents a Todo object.
+ */
+public class Todo extends Task {
+    /**
+     * Creates a todo.
+     * @param name A string representing the name of the todo.
+     * @param isCompleted A boolean value representing whether the todo has been completed.
+     */
+    protected Todo(String name, boolean isCompleted) {
+        super(name, isCompleted);
+    }
+
+    /**
+     * Creates a new Todo.
+     * The completion status is false by default.
+     * @param name A string representing the name of the new todo.
+     * @return The new todo created.
+     */
+    public static Todo newTodo(String name) {
+        return new Todo(name, false);
+    }
+
+    /**
+     * Creates a Todo using information in storage.
+     * The todo created is an already previously existing todo with its details recorded in storage.
+     * Usually called when starting up the application, to populate the TaskList.
+     * @param name A string representing the name of the existing todo.
+     * @param isCompleted A boolean value representing whether the todo has been completed.
+     * @return The existing todo created.
+     */
+    public static Todo existingTodo(String name, boolean isCompleted) {
```

Comment:
Maybe something like pullTodo(...)? I notice that your Javadocs describe the method really well. I referred to your Javadocs to see how I would name the method so maybe you can consider referring back to your Javadocs for method naming